test/framework/src/main/java/org/elasticsearch/test/rest/Stash.java

```diff
@@ -53,6 +55,12 @@ public void stashValue(String key, Object value) {
         }
     }
 
+    public void stashResponse(RestResponse response) throws IOException {
+        // TODO we can almost certainly save time by lazily evaluating the body
```

Comment:
This is just a thing I noticed while working on the REST tests - probably not important.